mods/accent-color-sync.wh.cpp

```diff
@@ -0,0 +1,392 @@
+// ==WindhawkMod==
+// @id                  accent-color-sync
+// @name                Accent color sync
+// @description         Syncs OpenGlass & StartIsBack++ colors with DWM
+// @version             1.0
+// @author              CatmanFan
+// @github              https://github.com/CatmanFan
+// @exclude             C:\Windows\system32\*

```

Comment:
Since only explorer.exe is included, this line has no effect and can be removed.